src/lib.rs

```diff
@@ -9,12 +9,9 @@
 extern crate gio_sys;
 extern crate glib_sys;
 extern crate gobject_sys;
-extern crate ostree_sys;
```

Comment:
If you are enjoying spring cleanups, I think we can directly drop all the `extern crate` entries here as the PR moves to 2018 edition.

Reply:
Yep, done!